spec.md

```diff
@@ -1076,7 +1081,7 @@ For success responses, the following fields are defined:
 | plan_id | string | The ID of the Service Plan from the catalog that is associated with the Service Instance. |
 | dashboard_url | string | The URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`9189kdfsk0vfnku` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the provision request, even if the service is provisioned asynchronously. |
 | parameters | object | Configuration parameters for the Service Instance. |
-
+| attributes | object | Attributes generated by the platform relevant to the Service Instance. This field will only have attributes generated by the platform and cannot be updated. |
```

Comment:
Attributes are generated by the broker. Right?

Reply:
that is correct @fmui

Reply:
I just fixed this sentence, thanks and good catch!